plugins/modules/kms_key.py

```diff
@@ -429,41 +444,47 @@
 from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
 from ansible_collections.amazon.aws.plugins.module_utils.core import is_boto3_error_code
 from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry
-from ansible_collections.amazon.aws.plugins.module_utils.ec2 import ansible_dict_to_boto3_tag_list
-from ansible_collections.amazon.aws.plugins.module_utils.ec2 import boto3_tag_list_to_ansible_dict
-from ansible_collections.amazon.aws.plugins.module_utils.ec2 import camel_dict_to_snake_dict
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import (
+    ansible_dict_to_boto3_tag_list,
+)
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import (
+    boto3_tag_list_to_ansible_dict,
+)
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import (
+    camel_dict_to_snake_dict,
+)
```

Comment:
These line breaks really don't add anything, please can we at least use a longer line length for black

Reply:
@jillr @goneri @alinabuzachis

Was there an internal decision that you're moving to using black?  Honestly some of these blind changes make things **less** readable rather than more.

Reply:
Not a formal decision for Python content, no.  We will be using ansible-lint for user-facing YAML content like roles though so there's some motivation to try to be more consistent in formatting generally.

Reply:
@tremble  Didn't we say on IRC that the most feasible solution is to apply back for new units or refactored module_utils only for the moment (until we come up with a strategy to apply black for the whole collection)?

Reply:
We mentioned that it's an option.  I didn't think anything had been agreed upon.  We certainly should look into what line length we use, the ~100 seems to be too short and results in things like the above.

Reply:
That's right, it wasn't something agreed upon, it was only a suggestion due to the massive number of PR adding unit tests and lots of refactoring for module_utils. However, I guess we should come up with a strategy first before applying consistent formatting.

Reply:
For this PR I am planning to undo the formatting done by black for the already existing file - plugins/modules/kms_key.py . I shall format the newly added unit test file with `black -l 160  <file>` (line length = 160). @tremble @jillr @alinabuzachis What is your view on that?